README.md

```diff
@@ -51,7 +51,7 @@ Use the `-help` flag with either the client or server to see a list of flags.
 | `-latencyPercentiles` | `50=10,100=100` | response latency percentile distribution in milliseconds. |
 | `-lengthPercentiles`  | `50=100,100=1000` | response body length percentile. |
 | `-errorRate`          | `0`       | The chance to return an error. |
-| `-metric-addr`        | `<none>`  | Address to use when serving the Prometheus `/metrics` endpoint. No metrics are served if unset. Format is `host:port` or `:port`. |
+| `-metricAddr`        | `<none>`  | Address to use when serving the Prometheus `/metrics` endpoint. No metrics are served if unset. Format is `host:port` or `:port`. |
```

Comment:
Totally trivial -- the column alignment is now off by one character on this line.